arangod/VocBase/Properties/PlanCollection.cpp

```diff
@@ -68,6 +85,76 @@ ResultT<PlanCollection> PlanCollection::fromCreateAPIBody(
   }
 }
 
+}  // namespace
+
+PlanCollection::DatabaseConfiguration::DatabaseConfiguration(
+    TRI_vocbase_t const& vocbase) {
+  auto& server = vocbase.server();
+  auto& cl = server.getFeature<ClusterFeature>();
+  auto& db = server.getFeature<DatabaseFeature>();
```

Comment:
I would like to get rid of `getFeature` and use explicit dependencies instead. Can we take the features as references instead?